contrib/plugins/albumartist_website.py

```diff
@@ -77,10 +77,12 @@ def website_add_track(self, album, track, artistId):
         if self.website_queue.append(artistId, (track, album)):
             host = config.setting["server_host"]
             port = config.setting["server_port"]
-            path = "/ws/2/%s/%s?inc=%s" % ('artist', artistId, 'url-rels')
+            path = "/ws/2/%s/%s" % ('artist', artistId)
+            queryargs = {"inc": "url-rels"}
             return album.tagger.xmlws.get(host, port, path,
                         partial(self.website_process, artistId),
-                        xml=True, priority=True, important=False)
+                                xml=True, priority=True, important=False,
+                                queryargs=queryargs)
 
```

Comment:
We will prolly remove contrib/plugins soon, in favor of the external project https://github.com/musicbrainz/picard-plugins
Fixing plugins isn't much an issue and we implemented a way to install/update plugins from the UI, next step will be to add an option to auto-update plugins.